rialto_airflow/dags/publish_orcid.py

```diff
@@ -26,7 +26,7 @@
 
 
 @dag(
-    # schedule=@weekly,
+    schedule="@weekly",
```

Comment:
despite the commented out example, it seems you need to quote that shortcut (or you can use full cron syntax) 

see https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dag-run.html for examples